packages/metal-incremental-dom/test/render/attributes.js

```diff
@@ -50,6 +50,20 @@ describe('attributes', function() {
 	});
 
 	describe('listeners', function() {
+		it('should register the method name inside the listener', function() {
+			class TestComponent extends Component {
+			}
+			TestComponent.prototype.handleClick = function() {};
+			component = new TestComponent();
+
+			let config = {
+				'data-on-click': 'handleClick'
```

Comment:
Yeah, @Robert-Frampton, you are right. It will not affect the test but I'm going to change it.